polaris-service/src/test/java/org/apache/polaris/service/task/TableCleanupTaskHandlerTest.java

```diff
@@ -97,22 +105,30 @@ public void testTableCleanup() throws IOException {
       assertThat(
               metaStoreManagerFactory
                   .getOrCreateMetaStoreManager(realmContext)
-                  .loadTasks(polarisCallContext, "test", 1)
+                  .loadTasks(polarisCallContext, "test", 2)
                   .getEntities())
-          .hasSize(1)
-          .satisfiesExactly(
+          .hasSize(2)
+          .satisfiesExactlyInAnyOrder(
```

Comment:
1. For the single metadata with single snapshot case, verify the ability to delete statistics file